src/content/app/tools/blast/components/blast-input-sequences/BlastInputSequences.scss

```diff
@@ -86,3 +86,10 @@
   font-weight: $light;
   margin-right: 14px;
 }
+
+.alertButton {
+  display: inline-block;
```

Comment:
Don't think we need `inline-block` here. Doesn't make any difference when with or without it.